src/commands/analyze.ts

```diff
@@ -57,8 +57,9 @@ export default defineCommand({
     const outDir =
       nuxt.options.nitro.output?.dir || join(nuxt.options.rootDir, '.output')
 
-    nuxt.options.build.analyze.filename =
-      nuxt.options.build.analyze.filename || join(analyzeDir, 'client.html')
+    nuxt.options.build.analyze = defu(nuxt.options.build.analyze, {
```

Comment:
@danielroe Was this only for fixing the type issue or also runtime safety?

Reply:
it was just for the type issue.

Reply:
We could fix it by typecast no? but anyway looks good for analaze internal use...